src/main/java/seedu/address/logic/Messages.java

```diff
@@ -50,4 +50,13 @@ public static String format(Person person) {
         return builder.toString();
     }
 
+    /**
+     * Formats the {@code ClaimAmount} for display to the user by converting from cents to dollar and cents.
+     * The formatting was generated using ChatGPT.
+     */
+    public static String formatClaimAmount(int claimAmount) {
+        int centsInADollar = 100;
+        return String.format("$%d.%02d", claimAmount / centsInADollar, claimAmount % centsInADollar);
```

Comment:
Left a comment about this in PR #95, not sure how to navigate potential conflicts though